modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java

```diff
@@ -349,34 +351,36 @@ private String _upgradeDDMStructureVersionDefinition(
 
 				JSONArray rows = _jsonFactory.createJSONArray();
 
-				List<String> nestedFieldNames = (List<String>)tuple.getObject(
-					_TUPLE_DDM_FORM_NESTED_FIELD_NAMES);
+				List<String> nestedDDMFormFieldNames =
+					(List<String>)ddmFormFieldTuple.getObject(
+						_DDM_FORM_FIELD_TUPLE_NESTED_FIELD_NAMES);
```

Comment:
@marcosapmf, inconsistent naming. `nestedDDMFormFieldNames` is receiving the value of `...NESTED_FIELD_NAMES`, maybe you should keep the name `nestedFieldNames`.